.github/workflows/main.yml

```
@@ -24,6 +24,8 @@ jobs:
 
       - name: Build
         run: npm run build
+        env:
+          BANNER_SERVER_BASE_URL: https://banner-server-mtls.cert.cfapps.sap.hana.ondemand.com
```

```suggestion
          BANNER_SERVER_BASE_URL: ${{vars.BANNER_SERVER_BASE_URL}}
```

Variable is already created.